src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -0,0 +1,235 @@
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<configurator>
+  <domain name="Energy Management"/>
+
+  <enum name="EvseStateEnum" type="ENUM8">
+    <cluster code="0x0099"/>
+    <item name="NotPluggedIn"         value="0x00"  optional="false"/>
+    <item name="PluggedInNoDemand"    value="0x01"  optional="false"/>
+    <item name="PluggedInDemand"      value="0x02"  optional="false"/>
+    <item name="PluggedInCharging"    value="0x03"  optional="false"/>
+    <item name="PluggedInDischarging" value="0x04"  optional="false"/>
+    <item name="SessionEnding"        value="0x05"  optional="false"/>
+    <item name="Fault"                value="0x06"  optional="false"/>
+  </enum>
+
+  <enum name="SupplyStateEnum" type="ENUM8">
+    <cluster code="0x0099"/>
+    <item name="Disabled"              value="0x00"  optional="false"/>
+    <item name="ChargingEnabled"       value="0x01"  optional="false"/>
+    <item name="DischargingEnabled"    value="0x02"  optional="true" />
+    <item name="DisabledError"         value="0x03"  optional="false"/>
+    <item name="DisabledDiagnostics"   value="0x04"  optional="false"/>
+  </enum>
+
+  <enum name="FaultStateEnum" type="ENUM8">
+    <cluster code="0x0099"/>
+    <item name="NoError"            value="0x00"  optional="false"/>
+    <item name="EvseMeterFailure"   value="0x01"  optional="false"/>
+    <item name="OverVoltage"        value="0x02"  optional="false" />
+    <item name="UnderVoltage"       value="0x03"  optional="false"/>
+    <item name="OverCurrent"        value="0x04"  optional="false"/>
+    <item name="ContactWetFailure"  value="0x05"  optional="false"/>
+    <item name="ContactDryFailure"  value="0x06"  optional="false"/>
+    <item name="GroundFault"        value="0x07"  optional="false"/>
+    <item name="PowerLoss"          value="0x08"  optional="false"/>
+    <item name="PowerQuality"       value="0x09"  optional="false"/>
+    <item name="PilotShortCircuit"  value="0x0A"  optional="false"/>
+    <item name="EmergencyStop"      value="0x0B"  optional="false"/>
+    <item name="EvDisconnected"     value="0x0C"  optional="false"/>
+    <item name="WrongPowerSupply"   value="0x0D"  optional="false"/>
+    <item name="LiveNeutralSwap"    value="0x0E"  optional="false"/>
+    <item name="OverTemperature"    value="0x0F"  optional="false"/>
+    <item name="Other"              value="0xFF"  optional="false"/>
+  </enum>
+
+  <bitmap name="TargetDayOfWeekBitmap" type="BITMAP8">
```

Comment:
This naming does not match what I see in the spec.  Though the spec calls it TargetDayOfWeekBitmap some places; just not in the actual type definition.  Spec needs fixing.

Also, `bitmap8`.

Reply:
Spec heading updated in spec [PR](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7813)
Changed to `bitmap8`